packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -139,35 +139,11 @@ class _CupertinoTextFieldSelectionGestureDetectorBuilder extends TextSelectionGe
 ///
 /// {@macro flutter.widgets.EditableText.onChanged}
 ///
-/// To control the text that is displayed in the text field, use the
-/// [controller]. For example, to set the initial value of the text field, use
-/// a [controller] that already contains some text such as:
+/// {@tool dartpad}
+/// This sample shows how to get a value from a CupertinoTextField via the [onSubmitted]
```

Comment:
It looks like this description doesn't match what the example does. I'd just keep the original description.

Reply:
That's my bad, restored it